carta/session.py

```diff
@@ -368,11 +368,13 @@ def open_image(self, path, hdu=""):
             The path to the image file, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
         hdu : {1}
             The HDU to select inside the file.
+        complex : {2}
+            Arithmetic expression to use if opening a complex-valued image. A member of :obj:`carta.constants.ArithmeticExpression` or ``None``. By default the image is assumed not to be complex.
```

Comment:
Don't put the type information in here -- it will be filled in automatically by the validation decorator where you have `{2}`. You can check this by building the docs locally.

Reply:
@confluence, I still have no idea how to use sphinx to build a doc for our python code. I promise I will do that this weekend.   Nevertheless, I would like to make sure what does **type information** mean here? Do you mean the sentence  of **A member of :obj:`carta.constants.ArithmeticExpression` or ``None``**?.

Reply:
make sure the theme is installed first
`pip install sphinx-rtd-theme`
then
`make html` in the docs folder.

Reply:
this is what it reads in the doc. you may refer to "colormap" docstring in the image class to see how it works as a references. 

![Screen Shot 2023-04-13 at 19 03 07](https://user-images.githubusercontent.com/20819712/231739582-68552ec2-6833-4c69-a7b0-3bd3606d247d.png)